app/src/main/res/layout/fragment_map.xml

```diff
@@ -43,13 +43,24 @@
             tools:ignore="RtlHardcoded"
             />
 
+
     </RelativeLayout>
 
     <com.google.android.gms.maps.MapView
         android:id="@+id/map"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
         />
+    <ImageView
+        android:layout_width="40dp"
+        android:layout_height="40dp"
+        app:layout_constraintEnd_toEndOf="parent"
+        android:layout_margin="10dp"
+        app:layout_constraintTop_toBottomOf="@id/relLayout1"
+        android:scaleType="centerCrop"
+        android:id="@+id/ic_gps"
+        android:src="@drawable/ic_gps"
+        />
```

Comment:
After testing the app:
This recenter button should use `android:tint="?colorOnSurface"` to ensure that it is also visible on dark maps.